modules/translate/main.js

```diff
@@ -78,6 +78,11 @@ module.exports = (bot, channel, user, args, id, message, extra) => {
         return channel.fetchMessages({ before: id, limit: 1 })
             .then(msg => translate(msg.first().content))
             .catch(error => util.error(error, "translate", channel));
+    }
+    else if (isNaN(query)){
```

Comment:
also, shouldn't you be checking for `!isNaN(query)`? why not just `Number(query)`?